homeassistant/components/fully_kiosk/services.py

```diff
@@ -114,7 +114,7 @@ async def async_set_config(call: ServiceCall) -> None:
                 {
                     vol.Required(ATTR_DEVICE_ID): cv.ensure_list,
                     vol.Required(ATTR_KEY): cv.string,
-                    vol.Required(ATTR_VALUE): vol.Any(str, bool, int),
+                    vol.Required(ATTR_VALUE): vol.Any(str, bool, int, object),
```

Comment:
So what kind of config can we set with this? How many options are there? Would it be logical to create a service per config?

Reply:
This lets you set any of the ~300 configuration options under the Settings tab in the Fully Kiosk app. Toolbars/appearance, screensaver settings, zoom/scaling, power settings, there's a ton of different options.